1001-alerts-design-guidance.md

```diff
@@ -0,0 +1,56 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-21
+updated: 2024-10-21
+---
+
+# Alert Design Guidance
+
+The [alert](https://clarity.design/documentation/alert) component displays banners that demand the user's attention for important messages.
+
+## Guidance
+
+### App-Level Alerts
+
+App-level alerts support only error, warning, and info statuses.
+
+- **Must** be used exclusively for the global context of an application.
+- **Must** be placed at the top of *all* content. 
+- **Must not** be used to convey the success of an operation.
+- Alerts for communicating operation success **must** appear in context or as a notification message.
+
+### Standard Alerts
+
+Standard alerts support error, warning, info, and success statuses.
+
+- **May** use lightweight alerts in places where vertical space is scarce and information density is needed; otherwise, the default size **should** be used.
+- **Must** be used for messages reflecting the context of the user in the application.
+
+
+### Lightweight Alerts
+
+- **Should** be used inside containers and other components and layouts.
+- Use a success alert to deliver a static, persistent success message.
+- Use an info alert to provide context-specific information to the user. Be cautious not to overuse or replace regular content.
```

Comment:
```suggestion
- Design teams **must** use an info alert to provide context-specific information to the user. This helps clarify actions or statuses without overwhelming the user with too much information, ensuring they remain focused on their tasks. However, it’s important to avoid overusing info alerts or replacing essential content with them, as this could lead to confusion or distraction.
```